test/runtests.jl

```diff
@@ -5,8 +5,10 @@ using Compat
 using BinDeps
 
 Pkg.add("Cairo")  # Tests apt-get code paths
+Pkg.build("Cairo")
```

Comment:
good call. I think we should actually move the `Pkg.add` lines to `test/REQUIRE` instead, so we make slightly fewer permanent changes to the users' package state by running the tests here